packages/sdks/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -825,6 +830,7 @@ class DescopeWc extends BaseDescopeWc {
 
             this.flowState.update({
               code: data.code,
+              exchangeError: data.exchangeError,
```

Comment:
this means that we may override code and exchange error together, we can split into two events as well but I think that is not required at the moment, because there is no case there will be both `exchangeError` and `code`

<img width="909" height="543" alt="image" src="https://github.com/user-attachments/assets/284e32b3-3601-442b-9b4e-d1d96545a869" />


@nirgur lmk if you think otherwise